Makefile

```diff
@@ -1,7 +1,7 @@
 CC ?= cc
 CFLAGS ?= -O2 -fPIC
 PREFIX ?= /usr/local
-VC ?= 0.1.17
+VC ?= $(shell git ls-remote https://github.com/vlang/vc.git --tags master | awk '{print $$1;}')
```

Comment:
@medvednikov
now Makefile will automatically update/use optimized latest version of vc and cache

Reply:
Is this valid?
```
$ git ls-remote https://github.com/vlang/vc.git --tags master | awk '{print $$1;}'
e0c9a842b3d034aa5e5a1e2ea8b22dc9464b7765  refs/heads/master
```

Reply:
It is. In Makefiles $$ => $, so the awk will see $1 .
![retrieve remote master commit hash inside Makefile](https://url4e.com/gyazo/images/3ffe313f.png)

Reply:
@whoizit In Makefile, `$$` is `$`

Reply:
A possible negative cost is that will force make to do a network operation on each make run, which may take +1,2 seconds.

Reply:
That network operation can be avoided by manually setting a value to VC.
`VC=0.1.17 make` or `VC=master make`.
But that will of course skip any updates as well.